test/unit/processing/siti-agri/handle-known-defects/recalculate-bps-penalties.test.js

```diff
@@ -0,0 +1,123 @@
+const { recalculateBPSPenalties } = require('../../../../../app/processing/siti-agri/handle-known-defects/cap-bps-penalties/recalculate-bps-penalties')
+const { P01, P02, P04, G00 } = require('../../../../../app/constants/line-descriptions')
```

Comment:
I think the constants should be defined above the { recalculateBPSPenalties } with a space to separate them